### PR TITLE
StorageCluster: log unknown error in deleteStorageClasses

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

**Comment by iamniting** [comment]
@obnoxxx @raghavendra-talur @jarrpa Can you pls add /ok-to-test and review the PR.

File: pkg/controller/storagecluster/uninstall_reconciler.go [hunk 1]
```diff
@@ -46,6 +46,8 @@ func (r *ReconcileStorageCluster) deleteStorageClasses(instance *ocsv1.StorageCl
 			}
 		case errors.IsNotFound(err):
 			reqLogger.Info(fmt.Sprintf("Uninstall: StorageClass %s not found, nothing to do", sc.Name))
+		default:
+			reqLogger.Info(fmt.Sprintf("Uninstall: Unexpected Error %v while getting StorageClass %s", err, sc.Name))
```
Comment (raghavendra-talur) [inline pkg/controller/storagecluster/uninstall_reconciler.go:50]: If I may suggest a change, it is better to keep error at the end of the sentence.

Comment (jarrpa) [reply pkg/controller/storagecluster/uninstall_reconciler.go:50]:
```suggestion
			reqLogger.Info(fmt.Sprintf("Uninstall: Error while getting StorageClass %s: %v", err, sc.Name))
```

Comment (raghavendra-talur) [reply pkg/controller/storagecluster/uninstall_reconciler.go:50]: @jarrpa I think the args are reversed.

Comment (jarrpa) [reply pkg/controller/storagecluster/uninstall_reconciler.go:50]: Updated

Comment (iamniting) [reply pkg/controller/storagecluster/uninstall_reconciler.go]: @raghavendra-talur @jarrpa Thanks for the suggestion, I have updated the first commit itself with the changes.

**Pull request merged**